hassio/dock/addon.py

```diff
@@ -100,7 +109,8 @@ def _run(self):
                 ports=self.addons_data.get_ports(self.addon),
                 devices=self.addons_data.get_devices(self.addon),
                 environment=self.environment,
-                volumes=self.volumes
+                volumes=self.volumes,
+                tmpfs=self.tmpfs
```

Comment:
Should I modify addons_data.get_tempfs to return a dict then ? I was thinking that the get_tmpfs should return a string since the parameter in config will be a string